.github/workflows/rust.yml

```diff
@@ -127,6 +126,37 @@ jobs:
           files: coverage.codecov
           flags: unittests
           fail_ci_if_error: false
+
+  integration_test:
```

Comment:
does it make sense to call this `query_integration_tests`? Since we may add other kinds of integration test in the future to sophon.